best-practices/README.md

```diff
@@ -183,7 +183,9 @@ Email
 JavaScript
 ----------
 
-* Use CoffeeScript.
+* Use ES6 when possible with a tool like [6to5].
```

Comment:
Ideally, a best practice won't change often (for example, "avoid global variables" and "keep classes small"). Will 6to5 stand the test of time? Or is it just a current solution?